roles/tendrl-storage-node/handlers/main.yml

```diff
@@ -10,3 +10,8 @@
   service:
     name=tendrl-node-monitoring
```

Comment:
I am not doing anything with tendrl-node-monitoring. This will go away after i resolve merge conflict. I will rebase and update the PR so that this will be corrected.